# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserChild.h
# Commit: 04d9dba72f22
# Full Hash: 04d9dba72f221b5808f077e3aac5522147008438
# Author: Jim Porter <jporter@mozilla.com>
# Date: 2019-06-04 21:42:40
# Regressor Bug: 1552017
# File Overlap Count: 1
# Description:
#   Bug 1552017 - Expand the kinds of URLs that can cancel content JS when navigating; r=smaug
#   
#   This patch makes several changes to the kinds of URLs where we can cancel
#   content JS when navigating between them:
#   
# ==============================================================================

diff -r 25b1ee5ee458 -r 04d9dba72f22 dom/ipc/BrowserChild.h
--- a/dom/ipc/BrowserChild.h	Tue Jun 04 16:44:29 2019 +0000
+++ b/dom/ipc/BrowserChild.h	Tue Jun 04 16:19:27 2019 +0000
@@ -827,9 +827,6 @@
                                        Maybe<WebProgressData>& aWebProgressData,
                                        RequestData& aRequestData);
 
-  nsresult CanCancelContentJSBetweenURIs(nsIURI* aFirstURI, nsIURI* aSecondURI,
-                                         bool* aCanCancel);
-
   class DelayedDeleteRunnable;
 
   TextureFactoryIdentifier mTextureFactoryIdentifier;